Formula/flif.rb

```diff
@@ -0,0 +1,75 @@
+class Flif < Formula
+  desc "Free Loseless Image Format"
+  homepage "http://flif.info/"
+  head "https://github.com/FLIF-hub/FLIF.git"
+
+  stable do
+    # patch 'install -s', see https://github.com/FLIF-hub/FLIF/issues/369
+    patch :DATA
```

Comment:
Can you please submit this patch to the upstream developers of this project as a PR and add a link to the upstream patch submission and explanation of why the patch is needed in a comment in the formula file. Thanks!